### PR TITLE
Update apply-sensitivity-label-automatically.md

### PR DESCRIPTION
**Pull request by edwinchan-microsoft**
@KCCross  / @chrfox

### CODE REVIEW

**Comment by KCCross** [comment]
Nice job! Suggested merge date is 8/9/22. Do Not Merge until then.

File: microsoft-365/compliance/apply-sensitivity-label-automatically.md [hunk 1]
```diff
@@ -99,6 +99,9 @@ Use the following table to help you identify the differences in behavior for the
 |Assign a Rights Management owner for emails sent from another organization |No |Yes|
 |For emails, replace existing label that has same or lower priority |No |Yes (configurable)|
 
+> [!NOTE]
+> (In Preview) Auto-labeling policies can use trainable classifiers as a condition to detect sensitive documents. You can define content with trainable classifiers in Exchange Online, Sharepoint Online sites, or OneDrive for Business accounts. For more information on Trainable Classifiers, see [Trainable Classifiers](classifier-learn-about.md).
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/apply-sensitivity-label-automatically.md:103]:
```suggestion
> (In Preview) Auto-labeling policies can use trainable classifiers as a condition to detect sensitive documents. You can define content with trainable classifiers in Exchange Online, SharePoint Online sites, or OneDrive for Business accounts. For more information on Trainable Classifiers, see [Trainable Classifiers](classifier-learn-about.md).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @edwinchan-microsoft, kindly commit the suggested change. Many thanks.

**Comment by cabailey** [comment]
As discussed, closing & replacing with 15503

**Pull request closed without merging**